junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/DefaultLauncher.java

```diff
@@ -69,6 +70,27 @@
 		return testEngines;
 	}
 
+	/**
+	 * @return {@code true} if the graph does _not_ contain a cycle; else {@code false}.
```

Comment:
I wonder if the `_not_` should be replaced with `<em>not</em>`?

Reply:
Good point.